test/README

```diff
@@ -3,33 +3,46 @@
 This directory contains automated integration tests for Cockpit, and the support
 files for them.
 
-To run the tests on Fedora, you need to install the following packages:
-
-    $ sudo yum install npm qemu-kvm python \
-         firewalld libvirt-daemon libvirt-client libvirt-python \
-         openssl expect rsync curl xz
+To run the tests on Fedora, refer to the [HACKING](../HACKING.md) guide for
+installation of all of the necessary build and test dependencies. There's
+no need to trigger a build manually - the test suite preparation step below
+will handle that.
 
 You may need to reboot after installing the above packages. For initialization
 the following command needs to be run as root to setup the libvirt bridged network:
 
     $ sudo ./test/vm-prep
 
+Note that this step also attempts to ensure nested virtualization is enabled,
+and may emit a spurious `rmmod: ERROR: Module kvm_amd is not currently loaded`
+warning when running on an Intel CPU (or vice-versa).
+
+If test failures are encountered that look like they may be related to problems
+with nested virtualization, refer to
+[this Fedora guide](https://fedoraproject.org/wiki/How_to_enable_nested_virtualization_in_KVM)
+for more details and recommendations on ensuring it is enabled correctly.
+
 ## Introduction
 
-To run the integration tests run the following. Don't run the integration tests
-as root:
+To build Cockpit and run the integration tests run the following (do NOT run
+the integration tests as root):
 
-    $ ./test/verify/run-tests
+    $ ./test/verify/run-tests --install
+
+The `--install` option indicates to (re)build and (re)install Cockpit before
+running the tests. Only code currently committed in the current branch will
+be installed and tested.
```

Comment:
I'm not sure I like that. I would like to keep documenting it as two separate steps. It's unnerving and absolutely unnecessary to rebuild the test VM (i. e. the `testsuite-prepare` part) when working on the tests, i. e. the installed cockpit doesn't change. It's of course fine to point out that running `run-tests` or any `check-*` with `--install` is a shortcut for this step.

(Already commened on this before, but it got hidden under "obsolete comment").